web/themes/custom/contribtracker/package.json

```diff
@@ -33,15 +42,15 @@
     "gulp-sass-glob": "^1.1.0",
     "gulp-svgstore": "^9.0.0",
     "gulp-typescript": "^6.0.0-alpha.1",
-    "postcss": "^8.4.40",
+    "postcss": "^8.4.38",
```

Comment:
@KosalaiV, why are you changing these dependencies?

Reply:
@hussainweb I haven't changed this, but we did rebase with main and got some conflicts in package.json. That could be the cause.
cc: @zeshanziya

Reply:
@KosalaiV It seems we accepted the current change instead of the incoming changes while resolving the conflict. Please revert it to its original version.